fearless/Modules/Staking/StakingRewardPayouts/ViewModel/StakingPayoutViewModelFactory.swift

```diff
@@ -77,23 +88,30 @@ final class StakingPayoutViewModelFactory: StakingPayoutViewModelFactoryProtocol
         return price
     }
 
-    private func daysLeftAttributedString(
+    private func timeLeftAttributedString(
         activeEra: EraIndex,
         payoutEra: EraIndex,
         historyDepth: UInt32,
+        eraCompletionTime: TimeInterval?,
         locale: Locale
     ) -> NSAttributedString {
         let eraDistance = historyDepth - (activeEra - payoutEra)
         let daysLeft = Int(eraDistance) / chain.erasPerDay
-        let daysLeftText = R.string.localizable
-            .stakingPayoutsDaysLeft(format: daysLeft, preferredLanguages: locale.rLanguages)
+        let timeLeftText: String = {
+            if daysLeft == 0, let eraCompletionTime = eraCompletionTime {
+                let formattedTime = (try? timeFormatter.string(from: eraCompletionTime)) ?? ""
```

Comment:
As i remember from our discussion we wanted to display "Payout expired" when time goes to zero.